src/components/GridSettings.js

```diff
@@ -0,0 +1,14 @@
+import { h } from '../resources/libs/preact.js'
+import { hiragana, katakana } from '../resources/jp/kanas.js'
+import { themedClass } from '../core/theme.js'
+import { getDifficultySubset } from '../core/logic.js'
+
+export default function GridSettings({ kind, difficulty }) {
```

Comment:
Please rename to something related to difficulty, like DifficultyGrid or something like that

Reply:
done